extensions/data-workspace/src/common/constants.ts

```diff
@@ -33,6 +33,7 @@ export const Select = localize('dataworkspace.select', "Select");
 export const WorkspaceFileExtension = '.code-workspace';
 export const DefaultInputWidth = '400px';
 export const DefaultButtonWidth = '80px';
+export const DataWorkspaceOutputChannel = localize('dataworkspace.outputChannel', "Data Workspace");
```

Comment:
I guess the channel name should be in English.

Reply:
so that in the instructions for logs you don't have to know what display language the customer is using.